desks-web-app/src/components/booking-components/WeekOverviewRow.vue

```diff
@@ -3,8 +3,8 @@
 
   <v-row class="mt-4">
     <v-col>
-      <h1 class="text-h4">{{ isPreviousWeekDisabled() ? "This Week" : "Week Overview" }}</h1>
-      <h3 class="text-subtitle-1">{{$format.date(calculateNthDayOfWeek(1))}} - {{$format.date(calculateNthDayOfWeek(5))}} | {{bookingDefaults?.room?.name}}</h3>
+      <h1 class="text-h4">Next Days Overview</h1>
```

Comment:
```suggestion
      <h1 class="text-h4">Desk Availability</h1>
```
Its only a suggestion but "Next days Overview" does not really fit when you on a page with dates further in the future.